crates/ln-dlc-node/src/tests/dlc_channel.rs

```diff
@@ -249,38 +251,126 @@ async fn can_open_and_force_close_channel() {
     // or similar
 }
 
-async fn setup_channel_with_position() -> (
-    Arc<Node<TenTenOneInMemoryStorage, InMemoryStore>>,
+#[tokio::test(flavor = "multi_thread")]
+#[ignore]
+async fn can_open_channel_with_min_inputs() {
```

Comment:
We should move this test to `rust-dlc` eventually, but our tests are easier to write for me.